src/ObjectOrientedStore.es6.js

```diff
@@ -155,6 +159,33 @@ export default class ObjectOrientedStore extends Store {
 							},
 
 							/**
+							 * Mock a public method and retain the
+							 * `this` context of the store which
+							 * has access to private variables
+							 * if you need them.
+							 *
+							 * @param object
+							 */
+							mockPublicMethod(object) {
+								each(object, (key, func) => {
+									if (publicMethods[key]) {
```

Comment:
throw an error in the `else` just to prevent any confusion from typos or functionality that we dont support